apps/src/code-studio/components/libraries/LibraryManagerDialog.jsx

```diff
@@ -116,11 +120,15 @@ export default class LibraryManagerDialog extends React.Component {
       );
     }
     return libraries.map(library => {
+      let onRefresh;
+      if (this.shouldRefresh(library)) {
```

Comment:
Why this always-false conditional instead of just removing the prop for now?

Reply:
Good point. Fixed.